toolchain/semantics/semantics_node.cpp

```diff
@@ -20,7 +20,7 @@ static auto PrintArgs(llvm::raw_ostream& out, std::pair<T0, T1> args) -> void {
   out << ", arg1: " << args.second;
 }
 
-void SemanticsNode::Print(llvm::raw_ostream& out) const {
+auto SemanticsNode::Print(llvm::raw_ostream& out) const -> void {
```

Comment:
I thought `clang-tidy` canonicalized the other way? Or is this `clang-tidy` at work?

Reply:
`clang-tidy` doesn't canonicalize this one in either direction, but this is the prevailing style in the toolchain (6 occurrences with leading `void` versus at least 146 with trailing `void`).

The prevailing style in explorer is the exact reverse (269 leading `void` versus 15 trailing `void`). :shrug:

Reply:
😭